project/src/components/sorting-element/sorting-element.tsx

```diff
@@ -0,0 +1,24 @@
+import { useAppDispatch, useAppSelector } from '../../hooks';
+import { changeSorting } from '../../store/action';
+
+type SortingElementProps = {
+  type: string;
+  toggle: () => void;
+}
+
+function SortingElement({type, toggle}: SortingElementProps): JSX.Element {
+
+  const dispatch = useAppDispatch();
+  const changeSortingType = (sortingType: string) => dispatch(changeSorting(sortingType));
+  const currentType = useAppSelector((state) => state.sortingType);
+
+  return (
+    <li
+      key={type} onClick={()=>{changeSortingType(type); toggle();}}
+      className={`places__option ${currentType===type&&'places__option--active'}`}
```

Comment:
```suggestion
      className={`places__option ${currentType === type && 'places__option--active'}`}
```

в переменную вынеси заодно. Ну и там фишка еще в том, что у тебя из-за твоего сравнение может в класс упасть false